tests/suite/layout/measure.typ

```diff
@@ -7,3 +7,10 @@
 }
 #text(10pt, f(6pt, 8pt))
 #text(20pt, f(13pt, 14pt))
+
+--- measure-given-area ---
+// Test `measure` given an area.
+#let text = lorem(100)
+
+#context measure(text)
+#context measure(width: 400pt, height: auto, text)
```

Comment:
This could probably be made into a compile-only test without an output image, similar to the test case above